unlock-app/next.config.js

```diff
@@ -71,6 +73,10 @@ module.exports = withTypescript({
       '/terms': { page: '/terms' },
       '/privacy': { page: '/privacy' },
       '/log': { page: '/log' },
+      '/post': { page: '/post' },
+      '/blog': { page: '/blog' },
     }
+
+    return addBlogPagesToNext(dir, pages)
```

Comment:
The above idea should also remove the need for this here. It can happen inside the `Blog` page startup, or even happen statically at build-time. Perhaps we can add a static blog generator similar to the paywall.min.js generator idea in a subsequent PR?